tests/helics/core/CoreFactory-tests.cpp

```diff
@@ -39,11 +39,11 @@ TEST(CoreFactory, ZmqCore)
 
 TEST(CoreFactory,MpiCore)
 {
-    EXPECT_EQ (helics::core::isCoreTypeAvailable (helics::CoreType::MPI), true);
-    auto core = helics::CoreFactory::create (helics::CoreType::MPI, "");
-    ASSERT_TRUE (core != nullptr);
-    helics::CoreFactory::unregisterCore (core->getIdentifier ());
-    core = nullptr;
+    EXPECT_EQ (helics::coretype::isCoreTypeAvailable (helics::CoreType::MPI), true);
```

Comment:
```suggestion
    EXPECT_EQ (helics::core::isCoreTypeAvailable (helics::CoreType::MPI), true);
```

I think this was supposed to stay the same?